examples/Enzyme/Library/Vector/VectorModel.cpp

```diff
@@ -39,13 +43,13 @@ void VectorModel::evalResidual()
 
 void VectorModel::evalJacobian()
 {
-  const int           n = x_.size();
+  const size_t        n = x_.size();
   std::vector<double> v(n);
   VectorModel         d_vector_model(n);
-  for (int idy = 0; idy < n; ++idy)
+  for (size_t idy = 0; idy < n; ++idy)
   {
     // Elementary vector for Jacobian-vector product
-    for (int idx = 0; idx < n; ++idx)
+    for (size_t idx = 0; idx < n; ++idx)
     {
       v[idx] = 0.0;
     }
```

Comment:
are we not able to use [`std::fill`](https://en.cppreference.com/w/cpp/algorithm/fill.html) to replace this loop? this just appears to be a zero-fill of the vector.